src/Components/UserInput.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import "bootstrap/dist/css/bootstrap.min.css";
+import Button from "react-bootstrap/Button";
+import Form from "react-bootstrap/Form";
+import InputGroup from "react-bootstrap/InputGroup";
+import Container from "react-bootstrap/Container";
+export default class UserInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      inputValue: "",
+    };
+  }
+
+  handleChange = (event) => {
+    this.setState({ inputValue: event.target.value });
+  };
+  handleSubmit = (event) => {
+    event.preventDefault();
+
+    if (this.state.inputValue) {
+      this.props.addItem(this.state.inputValue.toLowerCase());
+      this.setState({
+        inputValue: "",
+      });
+    }
+  };
+  render() {
+    return (
+      <div>
+        <Container className="col-12 col-lg-11 text-center">
+          <InputGroup id="main-input-field" className="mb-3 mt-4">
+            <Form.Control
```

Comment:
Not 100% sure here, but do we need to use a Form element here, when we actually are not using a form? Wouldn't a normal input suffice, to which we can apply bootstrap classes?